sway/config/output.c

```diff
@@ -204,6 +207,20 @@ void apply_output_config(struct output_config *oc, struct sway_container *output
 			execvp(cmd[0], cmd);
 		}
 	}
+	if (oc && oc->dpms_state != DPMS_IGNORE) {
+		switch(oc->dpms_state) {
+			case DPMS_ON:
```

Comment:
Style: deindent the case within the switch and add a space so `switch(` becomes `switch (`

Reply:
Done